horovod/keras/callbacks.py

```diff
@@ -152,3 +152,24 @@ def __init__(self, warmup_epochs=5, momentum_correction=True, steps_per_epoch=No
         """
         super(LearningRateWarmupCallback, self).__init__(K, warmup_epochs, momentum_correction,
                                                          steps_per_epoch, verbose, initial_lr)
+
+
+class ReturnedModelCheckpoint(keras.callbacks.ModelCheckpoint):
+    def __init__(self, filepath=None,
```

Comment:
I think we should not let the user specify `filepath`.  If they want a custom directory, they can just use the regular `ReturnedModelCheckpoint`.  Otherwise, there is no practical difference between this and `ModelCheckpoint`.

Reply:
sure